packages/staking/src/features/drawer/PoolDetailsCard.tsx

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Cardano } from '@cardano-sdk/core';
+import { Box, Card, ColorValueHex, ControlButton, Flex, Text } from '@lace/ui';
+import { useTranslation } from 'react-i18next';
+import { useOutsideHandles } from '../outside-handles-provider';
+import { useDelegationPortfolioStore } from '../store';
+import { PoolCard, PoolHr, PoolIndicator } from './StakePoolPreferences.css';
+import TrashIcon from './trash.svg';
+
+interface PoolDetailsCardProps {
+  poolId: Cardano.Cip17Pool['id'];
+  name: string;
+  index: number;
+  draftPortfolioLength: number;
+  colorSet: ColorValueHex[];
```

Comment:
Pass just a color here, please :wink:
```suggestion
  draftPortfolioLength: number;
  color: ColorValueHex;
```